ui/compontents.go

```diff
@@ -466,13 +466,13 @@ func (page *pageCommon) accountSelectorLayout(gtx layout.Context, callingPage, s
 	case callingPage == "send":
 		acctName := page.info.Wallets[page.wallAcctSelector.selectedSendWallet].Accounts[page.wallAcctSelector.selectedSendAccount].Name
 		walName := page.info.Wallets[page.wallAcctSelector.selectedSendWallet].Name
-		bal := page.info.Wallets[page.wallAcctSelector.selectedSendWallet].Accounts[page.wallAcctSelector.selectedSendAccount].TotalBalance
-		return d(gtx, acctName, walName, bal, page.wallAcctSelector.sendAccountBtn)
+		bal := page.info.Wallets[page.wallAcctSelector.selectedSendWallet].Accounts[page.wallAcctSelector.selectedSendAccount].SpendableBalance
+		return d(gtx, acctName, walName, dcrutil.Amount(bal).String(), page.wallAcctSelector.sendAccountBtn)
```

Comment:
set `page.info.Wallets[page.wallAcctSelector.selectedSendWallet]` to make the code line shorter and easier to read.